### PR TITLE
Updated Menu.yml per TIDOC-2376

### PR DESCRIPTION
**Pull request by jawa9000**
https://jira.appcelerator.org/browse/TIDOC-2376

### CODE REVIEW

File: apidoc/Titanium/Android/Menu.yml [hunk 1]
```diff
@@ -304,5 +304,5 @@ examples:
 
             // Ensure menu is displayed
             function doOpen(e) {
-                $.win.invalidateOptionsMenu();
+                $.win.invalidateOptionsMenu(); // should be --> $.win.activity.invalidateOptionsMenu();
```
Comment (sgtcoolguy) [inline apidoc/Titanium/Android/Menu.yml]: This doesn't look right

**Review by sgtcoolguy — CHANGES_REQUESTED** [review]
Looks like a bad insert here

**Comment by sgtcoolguy** [comment]
manually fixed and merged

File: apidoc/Titanium/Android/Menu.yml (hunk 1 above)
Comment (jawa9000) [reply apidoc/Titanium/Android/Menu.yml]: Thanks for the assist, Chris.

**Pull request closed without merging**